.idea/jarRepositories.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this file is not needed and can be replaced by references in build.gradle

Reply:
Okay, I see now this is generated by the latest version of intelliJ. This file can safely be ignored.